src/components/routing/routeProtectors/UserGuard.js

```diff
@@ -1,10 +1,26 @@
 import {Redirect} from "react-router-dom";
 import PropTypes from "prop-types";
 import Sidebar from "components/views/Sidebar";
+import {api} from 'helpers/api';
 
 export const UserGuard = props => {
-    let token = localStorage.getItem("token");
-    if (token && token !== 'undefined' && token != null && token !== '') {
+
+
+    // check if the token and the id match a user in the data base that is currently logged in
+    async function isTokenUserOnline() {
```

Comment:
Do you intend to overwrite the function in isTokenUserOnline.js? Or should this function here be deleted? [QUESTION]

Reply:
no, removing the userguard is another issue though, because it might come with some difficulty (or at least changes) of its own. so i did not focus on this one much. tbh, i probably forgot that i modified that file, because debugging took so long